middleware/allRoutes/handler.go

```diff
@@ -40,14 +39,21 @@ func Handler(routesHandler map[string]http.Handler, zebedeeClient *client.Client
 				return
 			}
 
-			// Construct contentPath with any colletion if present in cookie
+			if ok, err := regexp.MatchString(`\/latest$`, req.URL.String()); ok && err == nil {
```

Comment:
```suggestion
			if strings.HasSuffix(req.URL.String(), "/latest") {
```
hopefully, more efficient than regexp